### PR TITLE
Enable color for consistency checks diffs

### PR DESCRIPTION
**Pull request by silverwind**
Drone can display terminal colors, so force-enable it to make diffs more readable on it.

### CODE REVIEW

**Comment by silverwind** [comment]
Pushed a test commit so we can verify, will revert it later.

File: public/img/svg/octicon-diff-ignored.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg viewBox="0 0 16 16" class="svg octicon-diff-ignored" width="16" height="16" aria-hidden="true"><path d="M13.25 1c.966 0 1.75.784 1.75 1.75v10.5A1.75 1.75 0 0 1 13.25 15H2.75A1.75 1.75 0 0 1 1 13.25V2.75C1 1.784 1.784 1 2.75 1ZM2.75 2.5a.25.25 0 0 0-.25.25v10.5c0 .138.112.25.25.25h10.5a.25.25 0 0 0 .25-.25V2.75a.25.25 0 0 0-.25-.25Zm8.53 3.28-5.5 5.5a.749.749 0 0 1-1.275-.326.749.749 0 0 1 .215-.734l5.5-5.5a.751.751 0 0 1 1.042.018.751.751 0 0 1 .018 1.042Z"/></svg>
\ No newline at end of file
+<svg viewBox="0 0 16 16" class="svg octicon-diff-ignored testing-the-diff" width="16" height="16" aria-hidden="true"><path d="M13.25 1c.966 0 1.75.784 1.75 1.75v10.5A1.75 1.75 0 0 1 13.25 15H2.75A1.75 1.75 0 0 1 1 13.25V2.75C1 1.784 1.784 1 2.75 1ZM2.75 2.5a.25.25 0 0 0-.25.25v10.5c0 .138.112.25.25.25h10.5a.25.25 0 0 0 .25-.25V2.75a.25.25 0 0 0-.25-.25Zm8.53 3.28-5.5 5.5a.749.749 0 0 1-1.275-.326.749.749 0 0 1 .215-.734l5.5-5.5a.751.751 0 0 1 1.042.018.751.751 0 0 1 .018 1.042Z"/></svg>
```
Comment (delvh) [inline public/img/svg/octicon-diff-ignored.svg]: ?

**Comment by silverwind** [comment]
Color works:

image

There is no word highlight and that will require [additional tools](https://maximsmol.medium.com/improving-git-diffs-4519a6541cd1), but I'd say it's still an improvement over uncolored.

File: public/img/svg/octicon-diff-ignored.svg (hunk 1 above)
Comment (silverwind) [reply public/img/svg/octicon-diff-ignored.svg]: Was just a test as mentioned, reverting...

Comment (silverwind) [reply public/img/svg/octicon-diff-ignored.svg]: Reverted and force-pushed over it.

Comment (zeripath) [reply public/img/svg/octicon-diff-ignored.svg]: >Pushed a test commit so we can verify, will revert it later.

**Review by yardenshoham — APPROVED** [review]

**Review by delvh — APPROVED** [review]
Let's hope we don't have anyone who runs these `make` commands on a non-color capable tty…

**Comment by techknowlogick** [comment]
ping lg-tm

**Pull request merged**